force-app/main/default/classes/TestDataFactory.cls

```diff
@@ -68,11 +68,15 @@ public with sharing class TestDataFactory {
         Contact contact1 = new Contact();
         contact1.LastName = 'Test Contact';
         contact1.AccountId = accountRecord.Id;
+        contact1.NumPresentServiceDeliveries__c = 0;
```

Comment:
In a real org would these actually be zero or should they be null. In my mind if they do not have attendance records then by default they should be null. Is there a reason you needed to set these to 0?